tests/test_teacher.py

```diff
@@ -48,3 +48,45 @@ def test_GroundAtomsHold():
     assert len(response.holds) == 2
     assert response.holds[is_block_block]
     assert not response.holds[covers_block_target]
+
+
+def test_DemonstrationQuery():
```

Comment:
how's the speed of this test? (`pytest -s tests/test_teacher.py --durations=0`)

Reply:
pretty fast it seems; 0.01s